### PR TITLE
chore: 0.4.3-rc release

### PR DESCRIPTION
**Pull request by dnys1**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by ragingsquirrel3** [comment]
Should this change, too? https://github.com/aws-amplify/amplify-flutter/blob/main/packages/amplify/amplify_flutter/lib/src/method_channel_amplify.dart#L170

File: packages/amplify/amplify_flutter/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,10 @@
+## 0.4.3-rc (2022-03-31)
```
Comment (ragingsquirrel3) [inline packages/amplify/amplify_flutter/CHANGELOG.md:1]: nit: should be today's date

**Review by ragingsquirrel3 — APPROVED** [review]

File: packages/amplify/amplify_flutter/CHANGELOG.md (hunk 1 above)
Comment (dnys1) [reply packages/amplify/amplify_flutter/CHANGELOG.md]: Yep, thanks. Changed to Monday.

**Review by Jordan-Nelson — APPROVED** [review]

**Comment by HuiSF** [comment]
Should the RC version be `0.4.3-rc.1` to be consistent with the previous RC version scheme?

**Pull request merged**